PSReadLine/Render.cs

```diff
@@ -875,6 +876,26 @@ private static int LengthInBufferCells(char c)
             return 1 + (isWide ? 1 : 0);
         }
 
+        private static string SubstringByCells(string text, int countOfCells)
+        {
+            int length = 0;
+            int index = 0;
+            string substring = string.Empty;
```

Comment:
This variable seems unnecessary.

Reply:
Yes, I think I had something different in mind initially.